q2_demux/plugin_setup.py

```diff
@@ -125,7 +125,10 @@
     function=q2_demux.summarize,
     inputs={'data':
             SampleData[SequencesWithQuality | PairedEndSequencesWithQuality]},
-    parameters={'n': qiime2.plugin.Int},
+    parameters={
+      'n': qiime2.plugin.Int,
+      'delta': qiime2.plugin.Int
```

Comment:
Add the `Range` predicate (imported from the same place as `Int`) to both of these with `% Range(0, None)`